### PR TITLE
Implement try-out button (closes #813, #815, #816)

### LINKED ISSUE #813: Clear up which dependencies are needed for try-out button

### PR DESCRIPTION
**Pull request by guacke**
**Short description**
This PR will implement a try-out button, which instantiates a remote Jupyter notebook in which users can try GQCP.

**Related issues**
#813 #814 #815 #816

### CODE REVIEW

**Comment by guacke** [comment]
Although the Binder notebook currently launches, the kernel crashes on import of GQCP. This is because the Libint of conda-forge is loaded instead of the GQCG libint fork (which is currently disabled due to CI problems). As such, we need to

1. Fix the CI issues with Libint (https://github.com/GQCG-oss/libint/pull/3)
2. Re-enable the Conda workflow for GQCP.

**Comment by guacke** [comment]
Pre-merge notes:

- [ ] Uncomment ci-cd-conda
- [ ] Point notebook to develop

**Comment by guacke** [comment]
One test failing:

```
17/85 Test #17: Simple2DM_test .................................***Failed    0.06 sec
Running 3 test cases...
Will read basis set from $PREFIX/share/libint/2.4.2/basis/sto-3g.g94
$SRC_DIR/gqcp/tests/DensityMatrix/Simple2DM_test.cpp(137): error: in "two_electron_operator_expectation_value_different_orbital_bases": check std::abs(exp_val_AO - exp_val_random) < 1.0e-12 has failed

*** 1 failure is detected in the test module "Simple2DM"
```

Temporarily disabling tests for conda builds.

**Comment by guacke** [comment]
Current try-out link for binder: https://mybinder.org/v2/gh/GQCG/GQCP/feature/try-out?filepath=gqcpy%2Fexamples%2Ftry_it_out.ipynb

**Comment by guacke** [comment]
After merge, all examples/tutorial should be added to the README.md in `gqcpy/examples`.

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: gqcpenv
+channels:
+  - gqcg
+  - conda-forge
+  - intel
+  - defaults
+dependencies:
+  - python=3.7
+  - gqcp
+
```
Comment (xdvriend) [inline environment.yml:10]:
```suggestion
  
  
```

**Review by xdvriend — APPROVED** [review]
Awesome! I think it's a great idea to adjust all other examples. At the end we can maybe even add that `readme` to the website so people can use all of the examples we provide!

You missed a new line at the end of one file, other than that no remarks!

**Pull request merged**